acceptance/pipelines/install-pipelines/script

```diff
@@ -0,0 +1,25 @@
+tmpdir="./subdir"
+pipelines="$tmpdir/pipelines"
+mkdir -p $tmpdir
+
+title "install pipelines"
+trace errcode $CLI install-pipelines -d $tmpdir
```

Comment:
It's not obvious from the callsite what you're installing here.

Please rename to something more obvious, e.g. `install-pipelines-symlink` or `install-pipelines-cli`.